jdk/src/share/lib/security/java.security-linux

```diff
@@ -138,6 +139,7 @@ RestrictedSecurity.NSS.140-2.jce.provider.4 = com.sun.net.ssl.internal.ssl.Provi
 RestrictedSecurity.NSS.140-2.keystore.type = PKCS11
 RestrictedSecurity.NSS.140-2.javax.net.ssl.keyStore = NONE
 
+RestrictedSecurity.NSS.140-2.securerandom.provider = SunPKCS11-NSS-FIPS
```

Comment:
It's additional information that is available in all versions, but not 8. It is not really used anywhere at the moment, but I figured it would be nice to be consistent.

I can remove it, if you think it shouldn't be there.

Reply:
I think it should be removed: leaving it here would suggest it's something a user could configure, but it isn't used. Worse, if omitted, the profile would be rejected.

Reply:
Removed.